spec/models/transformation_mapping_item_spec.rb

```diff
@@ -100,4 +106,82 @@
       end
     end
   end
+
+  # ---------------------------------------------------------------------------
+  # Network Validation
+  # ---------------------------------------------------------------------------
+  context "Network validation" do
+    let(:ems_vmware) { FactoryBot.create(:ems_vmware) }
+    let(:vmware_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => ems_vmware) }
+
+    let(:ems_redhat) { FactoryBot.create(:ems_redhat) }
+    let(:redhat_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => ems_redhat) }
+
+    let(:ems_ops) { FactoryBot.create(:ems_openstack) }
```

Comment:
This could just be https://github.com/ManageIQ/manageiq/blob/2ae1efbb42de4776e8c4a10826454ffb7fc083fa/spec/models/transformation_mapping_item_spec.rb#L8

Reply:
Done.